packages/lib/src/components/Dropin/types.ts

```diff
@@ -95,14 +95,23 @@ export interface DropinConfiguration extends UIElementProps {
 }
 
 export interface onOrderCancelData {
-    order: Order;
+    order: {
+        orderData: string;
+        pspReference: string;
+    };
 }
 
 export interface DropinComponentProps extends DropinConfiguration {
     core: ICore;
     onCreateElements: any;
     onChange: (newState?: object) => void;
-    onOrderCancel?: (data: onOrderCancelData) => void;
+    onOrderCancel?: (
+        data: onOrderCancelData,
+        actions: {
+            resolve: (data: { amount: PaymentAmount }) => void;
+            reject: () => void;
```

Comment:
I believe the `reject ` must accept an `error?: string` as parameter in the function, as it seems that the merchant can reject the cancel order step with a custom error  by looking at the `getOnOrderCancel` implementation